scripts/oidc-provider.js

```diff
@@ -20,11 +20,14 @@ const config = {
   clients: [
     {
       client_id: 'testing',
-      redirect_uris: ['http://127.0.0.1:3000'],
+      redirect_uris: ['http://127.0.0.1:3000', 'http://localhost:3000'],
       token_endpoint_auth_method: 'none',
       grant_types: ['authorization_code', 'refresh_token']
     }
   ],
+  claims: {
+    org_id: null
+  },
```

Comment:
I couldn't see a better way to do this, but this config for `node-oidc-provider` allows it to return an `org_id` claim, by specifying the `claims` parameter to the authentication request. Ideally, I'd like the server to just echo back the `organization` value in the `org_id` claim but couldn't find a good way to do it or if that scenario is supported by the server. Makes sense if it's strictly OIDC compliant.

https://openid.net/specs/openid-connect-core-1_0.html#ClaimsParameter